package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sonarqube-scanner",
-  "version": "4.0.1",
+  "version": "4.1.0",
   "lockfileVersion": 3,
   "requires": true,
```

Comment:
I think we also need to bump the version line `9` otherwise it gets bumped every time we run npm install!

Reply:
good catch. I had to change it manually because `npm version` is taking my cygwin `git` binary and failing to commit :(